### PR TITLE
Add unit test for popover component

### PR DESCRIPTION
**Pull request by raimund-schluessler**
In preparation of the migration from `v-tooltip` to `floating-vue` in #2600, I wanted to write some unit tests for the affected components, such as `Popover`.

But no matter what I tried, I cannot get this popover to open, neither by default, clicking, or setting the prop after mounting. This PR contains some tests that should pass, but instead fail, because the popover does not open.

@nextcloud/vuejs If someone has an idea, what I am doing wrong here, I would be very grateful!

### CODE REVIEW

**Comment by skjnldsv** [comment]
> But no matter what I tried, I cannot get this popover to open, neither by default, clicking, or setting the prop after mounting

Maybe because its mountpoint is in body ?

File: tests/unit/components/Popover/Popover.spec.js [hunk 1]
```diff
@@ -42,7 +42,11 @@ describe('Popover.vue', () => {
 			expect(wrapper.find('.content').isVisible()).toBe(false)
 		})
 		it('is open when prop is set', async () => {
+			const div = document.createElement('div')
+			div.id = 'root'
+			document.body.appendChild(div)
 			const wrapper = mount(Popover, {
+				attachTo: '#root',
```
Comment (marcoambrosini) [inline tests/unit/components/Popover/Popover.spec.js:49]: shouldn't this be done via the `container` prop?

Comment (raimund-schluessler) [reply tests/unit/components/Popover/Popover.spec.js:49]: `attachTo` is an option from @vue\test-utils -> mount function, see https://v1.test-utils.vuejs.org/api/options.html#attachto It specifies to which HTMLElement the component will be mounted to.
`container` is an option of the `Popover` component itself, specifying where the popover menu will be attached to. The popover should open no matter the `container` prop (but it also doesn't open if I specify it).
I am a bit lost to be honest.

Comment (marcoambrosini) [reply tests/unit/components/Popover/Popover.spec.js:49]: 🤔

**Pull request closed without merging**